src/main/resources/templates/main.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <title>Shorty</title>
+    <link href="../static/node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet"
+          th:href="@{node_modules/bootstrap/dist/css/bootstrap.min.css}">
+</head>
+<body>
+<div class="container pt-3">
+    <h1 class="text-center">
```

Comment:
Please use 2 spaces to indent HTML Code. I will add an issue to this to add an editorconfig.